apps/android/app/src/main/java/chat/simplex/app/views/TerminalView.kt

```diff
@@ -1,49 +1,82 @@
+import androidx.compose.material.*
+import androidx.compose.runtime.*
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.text.AnnotatedString
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.font.FontFamily
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
 import androidx.navigation.NavController
 import chat.simplex.app.model.*
-import chat.simplex.app.views.chat.SendMsgView
+import chat.simplex.app.ui.theme.SimpleXTheme
+import chat.simplex.app.views.chat.*
+import chat.simplex.app.views.chat.item.ChatItemView
+import chat.simplex.app.views.helpers.CloseSheetBar
 import chat.simplex.app.views.helpers.withApi
+import com.google.accompanist.insets.ProvideWindowInsets
+import com.google.accompanist.insets.navigationBarsWithImePadding
 import kotlinx.coroutines.DelicateCoroutinesApi
+import kotlinx.coroutines.launch
+import java.time.format.DateTimeFormatter
 
 @DelicateCoroutinesApi
 @Composable
-fun TerminalView(chatModel: ChatModel, navController: NavController) {
-  Column {
-    Button(onClick = { navController.popBackStack() }) {
-      Text("Back")
+fun TerminalView(chatModel: ChatModel, nav: NavController) {
+  TerminalLayout(chatModel.terminalItems, nav::popBackStack, nav::navigate) { cmd ->
+    withApi {
+      // show "in progress"
+      chatModel.controller.sendCmd(CC.Console(cmd))
+      // hide "in progress"
     }
-    TerminalLog(chatModel.terminalItems, navController)
-    SendMsgView(sendMessage = { cmd ->
-      withApi {
-        // show "in progress"
-        chatModel.controller.sendCmd(CC.Console(cmd))
-        // hide "in progress"
+  }
+}
+
+@Composable
+fun TerminalLayout(terminalItems: List<TerminalItem> , close: () -> Unit, navigate: (String) -> Unit,
+                   sendCommand: (String) -> Unit) {
+  ProvideWindowInsets(windowInsetsAnimationsEnabled = true) {
+    Scaffold(
+      topBar = { CloseSheetBar(close) },
+      bottomBar = { SendMsgView(sendCommand) },
+      modifier = Modifier.navigationBarsWithImePadding()
+    ) { contentPadding ->
+      Box(
+        modifier = Modifier
+          .padding(contentPadding)
+          .fillMaxWidth()
+          .background(MaterialTheme.colors.background)
+      ) {
+        TerminalLog(terminalItems, navigate)
       }
-    })
+    }
   }
 }
 
 @Composable
-fun TerminalLog(terminalItems: List<TerminalItem>, navController: NavController) {
-  LazyColumn {
+fun TerminalLog(terminalItems: List<TerminalItem>, navigate: (String) -> Unit) {
+  val listState = rememberLazyListState()
+  val scope = rememberCoroutineScope()
+  val df = DateTimeFormatter.ofPattern("HH:mm:ss")
+  LazyColumn(state = listState) {
     items(terminalItems) { item ->
-      ClickableText(
-        AnnotatedString(item.label),
-        onClick = { navController.navigate("details/${item.id}") }
-      )
+        Text("${item.date.toString().subSequence(11, 19)} ${item.label}",
```

Comment:
DateTimeFormatter seems to be unused and `item.date.toString().subSequence(11, 19)` is somewhat hacky, wat the intention to use the formatter for the text?

Reply:
removed df, tacky will do in console